src/js/components/stock-movement-wizard/AddItemsPage.jsx

```diff
@@ -4,7 +4,7 @@ import { reduxForm, initialize, formValueSelector } from 'redux-form';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
-import validate from './validate';
+import { validate } from './validate';
```

Comment:
Can you explain what this does?

Reply:
This answer explains it the best: https://stackoverflow.com/a/36796281
Before this PR there was 'default export' in validate.js, and in this commit i made in this file 'named exports'

Reply:
And ... that's why I hate javascript.